index.bs

```diff
@@ -670,6 +673,8 @@ spec:css-syntax-3;
     The <dfn for="CredentialRequestOptions,CredentialCreationOptions">relevant credential interface objects</dfn> for a given
     {{CredentialCreationOptions}} or {{CredentialRequestOptions}} (|options|) is a set of [=interface objects=], collected as follows:
 
+    Note: This algorithm uses the [[#sctn-cred-type-registry|Credential Type Registry]].
+
```

Comment:
I don't think we actually need this note, but I don't object either.

Reply:
ok, thx, I prefer to keep it.